[WIP] Dog Borgs Mk. BORK BORK [WIP]

Aurorablade opened the pull request:
[WIP][WIP][WIP][WIP][WIP][WIP][WIP][WIP][WIP]
Please Direct all CONSTRUCTIVE feed back, donation of sprites, and etc in the comments.
Modules are subject to change and removal though i would love to keep oringal feel.

yes i did copy and paste the old change log cuase its not my work.

[WIP][WIP][WIP][WIP][WIP][WIP][WIP][WIP][WIP]

This adds three new borg modules, refered to as Dogborgs. They
look like giant dogs, and have a 64x32 sprite. These dogs currently come
in three varieties: Security, Medical, and Janitorial.

Features common to all dogs:

A pair of jaws which serves as a melee attack.
Secborgs and emagged mediborgs/janiborgs have a more powerful
version of this.
Boop Module
This allows them to detect reagents in a container, and detect any
oddities in the atmosphere around them (same as an atmospheric
scanner). They can also boop people.
Synthetic Tongue
This functions basically as a mop. You can clean up cleanable
decals, eat trash & energy cells for energy, and lick people's faces.
This can be converted into a stunbaton for emagged borgs.
Built-in sleepers.
These come in three variants for each dog module:
The Portabrig
The Medical Pod
The Garbage Compactor.
The medbelly acts as a sleeper, it stabilizes patients and allows
you to view their critical information. It can also inject things
like a sleeper can.
The portabrig functions similarly, but it is unable to inject
reagents.
The garbage compactor is able to consume both people One person at a time as well as items.
There is a maximum of 32 items in the compactor at any one time.
The sleepers have a built-in self-cleaning mode, which fills the
chamber with acid to destroy biological material and nonorganic
material alike. However, unless overridden with an cryptographic
sequencer, this is unable to be triggered while the sleeper is
occupied by a living being.
You can resist out of any of these on a 1 minute timer, so long as you aren't incapacitated (stunned/weakened/dying).
Module specific features:

k9 is the security model of dogborg.
Cyborg zipties.
10 force big dog jaws.
A stunbaton linked to it's internal power cell.
Police tape.
Cyborg disabler.
EMAG: Mounted laser.
Medihound is the medical model of dogborg.
Small jaws.
Health Analyzer.
Mini fire-extinguisher.
Borg hypo.
Syringe.
Beaker.
Custodial Hound/Scrubpup is the janitorial model of dogborg.
Light replacer which recharges in a charging station.
Holosign creator.
Cleans the turfs it walks over like a normal janitor borg.
Powergamer's wet dream section:

Small jaws do 5 damage when not emagged, 10 when emagged and toggled. Mediborg/Janiborg.
Large jaws do 10 damage. Secborg.
Emagged tongue has a 4 tick stun.
It takes almost exactly 2 minutes for an unarmored full-health human to expire to the cleaning cycle.
This includes a significant refactor to how cyborg modules are handled-
They are no longer hardcoded into a switch() in the pick modules
function, but rather are based on the module object itself. This
shouldn't break anything.

https://i.imgur.com/OoQaWnM.png
Gifs:
https://i.imgur.com/ThL1kFL.gifv
https://i.imgur.com/k8YZlv8.gifv

VOREStation/VOREStation#28
:cl:
add: Dog borgs! Bork some more! Medicla, Security, and Janitor flavored in stores now.
/:cl:

taukausanake commented on code/modules/mob/living/silicon/robot/dogborg/dogborg.dm:
```diff
@@ -60,7 +60,7 @@
 	modules += new /obj/item/weapon/reagent_containers/glass/beaker(src)//For holding the chemicals when the chemist is nice
 	..()
-	emag = null
+	emag = null //suggestion?
```
I think the null emag as it is was done to keep the mounted laser off? Sounds fair but if asking for a different emag module I can't think of any right now

taukausanake commented on code/modules/mob/living/silicon/robot/dogborg/dogborg.dm:
```diff
@@ -82,7 +82,7 @@
 	modules += new /obj/item/device/dogborg/sleeper/compactor(src)
 	..()
-	emag = null
+	emag = null //suggestion?
```
Same as above

Da-Dman234 commented:
... But why? Wasn't this just closed due to negative feedback?

taukausanake commented:
@Da-Dman234 No it was closed because Tiger thought no one wanted it and no one was treating it as an actual PR. At least the majority of people

@Aurorablade First and foremost get rid of the belly icon. That needs to be removed first. Then people can start being sensible a little maybe

Da-Dman234 commented:
Tiger thought no one wanted it because of the majority of people didn't. I highly doubt a new PR will change anything but who am I to stop you. I still suggest everything I did in the previous PR.

scrubmcnoob commented:
Make sure that belly is resprited.

Regen1 commented:
I am heavily against this, so let's break up why.

Modules are childish, cringy and doesn't fit paradise.
"belly, boop module, tongue" needs to be renamed, or changed to not be here at all.

The models originally are from Vorestation, they have a purpose of being part of a fetish that I am extremely uncomfortable with, and it should be noted that I am not alone in this, and it **WILL** make us be associated with Vorestation. Their purpose doesn't matter, it's from vorestation and it's a **bad** association which I do not want (No hard feelings towards you vorestation folks, you're all lovely people)

The sleeper function is literally the dog eating them, see ^

The 64x32 sprites doesn't work in general, it's not what ss13 is designed for and the models look weird with that scale.

There is no clear need for these borgs, they're added under the category of "Other", as to say, they serve no other purpose. They don't really shake up the meta, they're not extremely interesting or interactive, and they're not something that's well designed to fit into our existing community.

As of this PR right now I am absolutely 100% against it, the sprites **needs** to be changed, and any and all association with a dog vore fetish must be completely gone.

taukausanake commented:
Maybe some of these things could be changed from belly storage. I could imagine the medidog to have a back mounted sleeper instead. Give the secdog a better/faster moving grab speed. I'd like the janidog to still eat people. A great way to dispose of people but that's going to be argued always...

Aurorablade commented:
@taukausanake @scrubmcnoob @Regen1 need to find me a spriter.

also on the emag, those modules had to do with the compactor/medical pod. I am kinda an equalizer, its not right for the sec hound to have an emag but these don't..so i am wondering what that could be.

@Da-Dman234 
point on the borgs. they should be niche, Mediborgs are kinda Search and rescue orinited. Its why modules can be changed added or removed per suggestion to fit flavor and niche of the dogbork in question (i will never stop calling them dogbork)

I would like to keep the janihound. IT will just be a matter of changing things up, clean on walk being a toggle or take charge? Tongue clean only? I am open.

i don't know about the karma class for sec borgs, theres got to be a flavorful way to meet concerns here.

Please note the medibelly was renamed Medical Pod, I kinda like the portabrig and janitoral ones.. But again..find me a spriter with some ideas... *demanding clap*

taukausanake commented:
@Aurorablade If no one else does I'll try to make free time to mess around with it. Just need a portable install of Paint.NET to take to work

SamHPurp commented:
My feedback remains unchanged, so I'll re-include it:

----

Two tile wide sprites look fucking awful, and behave even worse code/gameplay wise. Remember all the outrage about the growth serum? Its that times 100.

Adding vore-dogs is going to do wonders for public perception of a 'furry station' (rightfully or wrongfully). Now while that isn't a problem in and of itself, ignoring the public image is a mistake a lot of people are making right now, its in vogue. Do at least give half a mind to it.

These vore 'dogs' eat players, **swelling their stomach's sprite**, again something vore fetishists adore. The original PR author, and maintainer, have said that the point of this PR is entirely the eating of players. Someone who very recently spent ages getting merge-ready a full on vore system. This was clearly done to gauge opinion, and since being elevated to maintainer have decided to push further with it.

Obvious conflicts with the Peacekeeper PR's intention of 'removing sec borgs', which is entirely un-addressed, and if anything makes things worse?

The animations look so out of place with the entire rest of the server. Just go watch the gifs.

There is still a huge amount of balance tweaks and more importantly situational differentiation between why you'd pick a sec borg, vs this vore animal borg, which has not been clearly defined in any way in the PR. Looking at the modules, I cannot see a single reason I'd pick the regular borgs over these clearly upgraded borgs.

There have been recent PRs / annoyance that fat people can eat people... so now we want to add even more vore mechanics?

----

This culminates in the fact that if this is indeed the direction the server wishes to take, understand that its a stepchange that may cause a further hemorrhaging of players, degradation of public opinion, and actively promoting sexual fetishes that many consider perverse.

Aurorablade commented:
@samhpurp Find me spriter. Suggest module changes to fill a niche.

ZomgPonies commented:
👍 To Peacekeeper borgs, who fit an actual niche and use. 👎 to this PR, because it's just not worth the perception hassles for something that is redundant to the server.

Aurorablade commented:
I feel its worth the hassle and wish to see it through i would however need a spriter for some things. However we cannot avoid all matters of perception no matter what we do.

SamHPurp commented:
You identify a need, gap and niche **first**, then develop the solution. The way you are coming at this is all wrong from a design perspective ignoring the proposal for a second.

What point does this serve?

scrubmcnoob commented:
I wouldn't rely and expect others to find the spriter. If you wish to see this in, find the spriter yourself.
Just letting you know as the people less enthusiastic about this PR aren't going bother and don't want this to sit in the void forever waiting for some 'spriter' to come and redo them

ProperPants commented:
Vore dog robots are depraved. The thought truly sickens me. 👎

Aurorablade commented:
as a note i am not telling people argueing against it to find me a better spriter to be hostile..but serioulsy..help.

@samhpurp That may be i am sometimes not the best at design. I feel it serves as a unique departure from the regular borg set. Thus we can tweak the modules and purposes they have.

I am going to admit i am not creative wording wise..so i am open to suggestions on how things could be worded for module actions as long as it kept some flavor of the borgs. No i do not mean the codebase of where its from. I would actually ask people to look passed that on a new feature we could make interesting.
Its not where its from its what we can do with it.

ProperPants commented:
![2017-06-18 18_27_07-removes all bestiality fetish material from the _tg_station codebase by iamgoof](https://user-images.githubusercontent.com/16659022/27264762-06971750-5454-11e7-802e-2a340578d6d1.png)

Qustinnus commented:
everyday we stray further from god’s light 


Now instead of you making us give arguments as to why this PR is bad, which a lot of people have done in the old and are doing in the current PR, why don't you give us a few arguments as to why this PR is a positive addition to any codebase ever.

I'm literaly trying to get interested in Para but every time I see shit like this and I get scared off back to other servers, because I think stuff like this lowers Para's RP levels lower than bloody Yog and Hippie.

i t  e a t s  y o u

Aurorablade commented:
Current ideas listing here so i can keep track after talking with others:
Rename BOOP module..either may a new acronym for it or other.
BOOP retains functionality for medi and jani.
Sec Dog BOOP/Nose acts as a forensic scanner for when the detective if off drinking..i mean what?
Make sec dog jaws vice grip/grab someone to detain (until another borg or officer can get there or they can be put in porta brig..though prota brig might be resistable). Adjust modules to compensate for new functionality.
Sec dog tracking module? Or too op? it would need a foil.

Portabrig and medical pod can look like back mounted sleepers but the sprite would need to look natrual on the borg (don't let me coder sprite it...please no)

Make the tongue thing at least FEEL robotic then organic in nature. (HAHA feel..)

swordsith commented:
https://www.youtube.com/watch?v=6SxjBWJPf78

LightFire53 commented:
Oh for the love of...

Will people not get it through their heads that 1. The sprite is being changes to not have an organic stomach, and 2. Due to this, it's not eating people.

With that said, the mounted sleeper is viable for medidogs. Sec dogs, could maybe deploy a bed or something instead of holding someone in a sleeper style unit. Jani dogs shouldn't be able to eat people, but have some form of acidic spray if emagged.

Additionally, I would like to see these have less than regular borgs, but maybe have a trade off of a slightly faster speed than borgs.

And one last thought for sec dogs: maybe have a Bola style option where they don't stun, but latch on to someone and slow them down?

ZomgPonies commented:
"Being changed" Do you have a spriter working on it? Because from the last comment I can see, you were asking the people against the PR to do your work for you to make it palatable, which is a ridiculous way of going about doing things. 

Remove the vore sprites.
Remove the vore references.
Balance the PR against secborgs, peacekeeperborgs and other borg modules.

Then present your work as original content and MAYBE people won't be so quick to associate it to the fetish. The onus is on the people wanting to get this merge to convince everyone else.

Aurorablade commented:
@zomgponies I said that is not how i meant it to sound. IT will be changed. IT SERIOUSLY WAS NOT and i thought did apologize if not i am doing so now. I just don't know as many spriters as some people do..i will coder sprite if needed.

While not the best way i would rather lay out the content as it is then reiterate it. The current doghound sprites are fine. if i have to remove thier run animations i will. They may need to be brought down to be uniform but that is byond my ability to codersprite and the clipping isn't that bad.

GeneralGoose commented:
@taukausanake  remove bellies? but people are ok with the fact theres already a feature that allows people to eat eachother? how does that work
@ProperPants oh shit im a meme now

karmapay2win commented:
You guys are being trolled. The guy just copy pasted the previous PR description without even bothering to copy the formatting.

mattroks101 commented:
It was funny the first time but now it's getting redundant.

Qustinnus commented:
even if you change the sprites, this is still a bad PR that adds nothing but bad association that a lot of Paradise players would still like to avoid. Honestly why even make the PR if you haven't done anything but copy the old one? I'm honestly doubting your ability to code all the stuff you're talking about and the sprites seem even more doubtful.

Honestly you're avoiding every argument ever made in this PR so I'm just going to assume this is a well structured shitpost.

Cactus2388 commented:
Some people actually get off to inflation/dog stuff that's in this PR. I'm not going to be associated with this shit.

Aurorablade commented:
The only trolls here are the last four poster. and i request if you don't have anything contructive to post then you not post. Thank you.

Cactus2388 commented:
I'm not trolling, people ACTUALLY get off to this stuff, you do know what "Public Reputation" is right? Also, calling people "Trolls" for genuinely not wanting something in the code base is a shit mentality for taking criticism.

LightFire53 commented:
I'm really trying to figure out why people are so opposed.

>is vore
We're removing that. Is there another reason?
>is vore

This isn't meant to discredit those who have proposed other issues, such as the art/animation style, balance issues, niche/utility, etc...

Cactus2388 commented:
I mean, do we really.need more borgs, or do we just want 2d robit spess dogs? (If this gets merged we can't bitch about not being a furry station)

mrturkeytoe commented:
I personally welcome our vorehound overlords, commence the swollen synthbellies. Also the dildo and vore PRs were jokes. I'm not sure if Tiger's vorehound one was, I personally find the idea of a mediborg with a built in sleeper neat.

Qustinnus commented:
@aurorablade

how am I a troll for voicing my opinion?  I literaly asked you twice to give me a good argument as to why this feature adds anything to the codebase and you still haven't responded.

@lightfire55
All the mechanics are still vore, the PR is still vore. Nothing was changed between the old PR and the new, besides vague promises.

You are still porting a feature from vorestation (nothing against vore as a community) which is associated with (you mightve guessed it) the vore fetish. It's in bad taste and literaly associates the entire PR with the fetish. The balance of the modules is crap too, and basically most of the things in this feature make it so unfitting for Para that you might as well make o r i g i n a l c o n t e n t instead of porting this from VORE.

LightFire53 commented:
@Qustinnus >All the mechanics are still vore, the PR is still vore.

I mean, I could post the definition of vore, but I feeling people understand what it is already. So what makes this vore, if we were to change it so the medidog has a sleeper, the secdog some other restraint/slowdown effect, and removing the idea from the janidog all together? I'm clearly missing something.

Qustinnus commented:
vore as in the server. not the fetish.

Either way, you're avoiding my original question of; what does this add.

Why does it specifically need to be a dog? a borg with a sleeper, sure. but why a dog? If you plan on replacing the sprites there is zero reason to keep that sprite and once again you are avoiding all questions regarding why this PR is so interesting in the first place.

Spacemanspark commented:
Robotic K-9 units? Sure. It's a robot dog, not necessarily furry. The concept is fine...
But I think this is the wrong way to do it. The association alone, as Regens said, is quite a concern to me.
Remove all vore references and make it a more original concept that fits a special niche.
@mrturkeytoe no

Aurorablade commented:
i am gonna work on it. So bear with me.

SamHPurp commented:
@Aurorablade Aside from all my previous criticism of the idea - people are upset that you opened a new PR with the exact same stuff without any changes for people to critique.

If you had gone away, worked on this, made changes based upon feedback, then re-submitted a new PR for these dog-borgs, there would be a lot more sensible commentary... potentially still negative... but a lot of the annoyance I'm detecting is largely down to not developing it at all since the most recent batch of feedback, and just re-posting it. But I could be wrong.

Does that make sense?

Aurorablade commented:
yes it makes sense..but i am scared to close it to reopen it again later cuase its gonna be memed hard. and i do like the feedback on active PRs. I pretty much just pulled tigers code to make it easier on me to build off of.

Aurorablade commented:
cause in essence..it is tigers code.

Qustinnus commented:
no in essence it's VORE code.

uraniummeltdown commented:
IMO the game doesn't need any more animal junk, half the races are just animals with a stupid name (unathi, vulp, taj, vox, kidan). I'd rather we leave borgs as humanoid/inanimate object robots rather than have animal robots too. Especially with robot dogs, people will just spout the doggo pupper meme crap rather than treat them like actual robots

SheepFighter commented:
>Tigercat makes a PR adding dildos. It's treated as an obvious joke.

>Tigercat make an April 1st PR porting Virgo's titular vore-code. Ditto.

>Tigercat makes a PR porting canine vore-cyborgs (hereafter "Vorgs.") "WTF, why do you think this is a joke? Why are you convinced it's a vore-vehicle? Get your mind out of the gutter!"

I'm fine with discussing this in a calm, mature manner, but I wasn't born yesterday. If you just want Power Rangers-esque mecha-dogs, that's reasonable. But that's not what these are. They're a straight up port of a fetish-server's fetishistic feature. Labeling this WIP does nothing to change that fact. If you don't want folks associating this with vore, make a version that lacks the digestive elements, instead of (as Tiger did) claiming that "The ability to transport people around is intended to be the main reason why anyone would ever use these over the normal borgs."

tigercat2000 commented:
@SheepFighter 
>instead of (as Tiger did) claiming that "The ability to transport people around is intended to be the main reason why anyone would ever use these over the normal borgs."

My intention _was_ to base their entire gimmick around being able to transport people. The initial reaction I got to the port didn't include the freaking out over the fact that their bellies bulge and sway- I didn't really notice it myself until it was pointed out later.

taukausanake commented:
I'm with @tigercat2000 here. I thought at first, "Oh cool so they can each carry a mob if differently each. Nice." Then I saw it was from VORE Station and though, "Oh well they still look awesome and functional. There's no problem with that." Then I saw the belly and though, "Yeaaaah that has to get changed. Literally no one will like that." Still think these are okay after changes, I can't understand why no one else thinks so. It's not like you have to port something in its entirety, that's basically every SS13 server out there

GeneralGoose commented:
honetsly in your current state reputation is the last thing you should be worried about guys

shazbot194 commented:
If we don't start worrying about our reputation at some point, we may see a lot of problems, such as attracting the wrong people to the server, which I say we might have already started, see Powergamers.

Vivalas commented:
Can we just... please not. The fact this is not a joke both annoys and worries me. I really don't like the idea of carrying around mobs anyways inside borgs despite the obvious vore implications here.

At the very least make it not a dog, and more like a robot. Make it fit with the theme of other borgs, maybe a cart with robot eyes or something of the sort, but not this. And it needs downsides too, otherwise this becomes secborg / medborg 2.0.

Bram961 commented:
As for the vore association I would like to stay as far away as possible. The flak we would get would be massive.
Judging the PR on the concept of carrying people. I feel like having borgs that could carry people, essentially forming a protective shell around someone is a bad idea.
Borgs can already be tough to take out so I can already see the new meta for traitors and captain during nukeops riding around in their borg chariot letting their borg take all of the damage until it dies at which point they run off unharmed. Generally a feature like that should be reserved for  things that arent mass produceable as soon as there is enough metal and glass and a bit harder to get like the oddyseus.

outbools commented:
Sigh... can we stop discussing that thing already? Are you really that desperate for new borgs? Is there something that current para drones can't do except your urge of canine roleplaying?

Aurorablade commented:
@bram961 good thing theres not a syndie module then. We can try to think of things to try to balance it. right now i am trying to hash out vice jaws.

SheepFighter commented:
@taukausanake Yeah, but you didn't go to the trouble of creating an elaborate meme PR porting vorecode. I find it *highly* implausible that Tiger didn't know what he was getting into with this. He went to all the trouble of porting vorgs from VORE but just sort of missed the belly-bit? Yeah...

At this point I'm utterly convinced he's trolling, so this will be my last word on the matter.

outbools commented on code/modules/mob/living/silicon/robot/dogborg/dog_sleeper.dm:
```diff
@@ -0,0 +1,454 @@
+//Sleeper
```
Gee, look. Isn't it 454 new shiny lines of Odysseus mounted sleeper code made into freaking vore fetish?

Aurorablade commented:
@Bram961 as some ideas for sleepers (and i di reach out to a spriter for some help just got to hear back), Take damage, chance to be ejected, Take Weapon damage chance to be VIOLENTLY ejected. Grab intent on borg, yank out of sleeper (and if they borg isn't emagged or on some odd lawset they have to comply). Death you ARE violently ejected.

Birdtalon commented:
This only thing this would remotely fly with me in any way shape or form would be to wipe the slate clean and start again from a blank canvas. You could make as many changes and tweaks and re sprites as you want to but you - however the community is slow to forget and this will forever be associated with a fetish which many people, including myself are incredibly uncomfortable with. 

I am not going to make any further comments on anything else in this PR because even if the mechanics  and sprites were the best in the world I would still be against it for that reason alone.

To quote Oblivion: Not now, not later, not ever.

Aracino commented:
I never log in. But I will log in to say please for the love of god no. The server doesn't need more borgs and especially something from a fetish server. This really doesn't fit the theme, and the only dog borg I want in ss13 is a E-N.

Bram961 commented:
I don't see the merit of this either besides just adding dogborgs for the sake of dogborgs. We have heared the arguments against them. What are the arguments for them?

Ralta commented:
Not gonna read all the comments, not gonna comment on utility, just gonna say those animations are fucking GORGEOUS.

Draconic-saint commented:
Without looking at the macroclimate of ss13 and the public relations stuff, this just doesn't...fit. And that's coming from the guy who thought ( and still thinks) that one tile sized ww2 tanks were accaptable to be added in.

Alot of the stuff stated above holds my opinion:

- 64x32 is not something we want, look at pinetree and puma sprites, they just clip and break and all kinds of horrible stuff happens.

- Even without the vore stuff, sleepers should not be a borg thing, IMO. The tongue and jaws just have me wondering "why?". And when we DO consider the association with vore it just gets worse from there. Combined with the tonguestuff it's just weird and brings associations we've been trying to combat/avoid for a long time.

For the jaws/biting: Secborgs have tasers, flashes and batons, no need for more combat modules. Mediborgs and Janiborgs don't need modules like that to begin with.

The only way I'd ever support anything remotely like this would be custom borg sprites, depending on how the sprites look. No custom modules of the kind proposed here,

outbools commented:
@Draconic-saint xenoarch when

Draconic-saint commented:
@outbools No idea, and that ain't what this PR is about

Necaladun commented:
@outbools Keep this on topic.

Everyone in general: Keep this civil and obey rule 0.

Ssirius commented:
I think, I think its time to stop. 
It does not fit the server, it does not fit the theme, most of the load outs they have are straight up upgrades over normal borgs (that are already in some cases quite unbalanced). 
The only reason to have them in is painfully obvious, and if that is what people that want them are after there are servers that do that kind of thing way better Paradise ever will.

I'm so opposed to it my figurative corpse jumped up from the grave to write this.

ExitGame commented:
I don't know why you will be looking for spriters if there are practically no volunteers if there is heavy opposition from both staff and playerbase to this PR.

Necaladun commented:
People can desire a new feature for the server without it being a fetish thing. ERP is and always will be banned. This won't be being merged while they still retain the vore aspects. That is not why they're being proposed. What is "painfully obvious" is in your head, not the originator.

This is a WIP so we can discuss changes that would make them balanced in comparison to normal borgs.
Saying they're unbalanced as they are has been acknowledged, and discussion on how to balance them is requested. Simply saying they're unbalanced is unhelpful.

Sci-fi from WH40K to Dr Who has various forms of robotic dogs. I don't see why this is particularly out of theme for the server.

Birdtalon commented:
I get the feeling that discussion of adding "dogborgs" as a proposal would be better suited to our forums rather than a PR which as it stands has made few changes to differentiate these additions from their VORE origins so far. If discussion is required on the matter; time has shown that the Github is not the best place for a reasonable discussion of something such as this.

I can guarantee that after the initial "flurry" of comments and memes after the PR is opened that discussion will stagnate. @Necaladun has just stated above this post that this will not be merged while still containing fetishist aspects and I believe that the whole concept would need a complete rethink in order to shed that particular trait.

ExitGame commented:
@Necaladun
This PR doesn't even need a change, the entire concept here should be dumped and a entirely new concept should be made instead, what is even the difference? They don't even fill any "niche" since all of that can be accomplished by just cusomizing your actual character when being a Security Officer.
Having this PR up to discuss change serves no purpose as it won't be done, the only "change" thats being spoken out here is to just trash the whole idea and if anybody wants to think up a entirely new one similar to this.

And dogborgs definitely have no place on Paradise, I always thougt Staff pushed for a higher RP level supposed to be set in a professional research facility and that we have to be the change that we want to be, but throwing such PRs at the playerbase does the opposite of helping them with RP because this is just cringey silliness.

SamHPurp commented:
I think we've done to death the reasons against this. The onus is now on the author to drastically improve, or close.

Necaladun commented:
I think it's fair enough to give them a few days or weeks to take in peoples feedback of how to balance and change it, code that, then review that. If Fethas wants to leave it open on the github to do that, I see no reason why not.

Ssirius commented:
@Necaladun I did not come here to argue, simply to state my opinion, that **> I <** do not like it and do not want it, and that I feel like it wouldn't fit the theme, the balance, the look, in any way shape or form even if the sprite gets reworked, just removing the bloated silicone person carrying round belly would not be enough.

> What is "painfully obvious" is in your head, not the originator.

Are we looking at the same PR? Barring the sprite with or without the belly, boop modules, big dog jaws, synthetic tongues, perhaps only my mind is in the gutter, but a lot of other people got the same impression as I did, coincidence perhaps.
Just take a look at E-N, does his sprite look anything like these sprites do.

> This is a WIP so we can discuss changes that would make them balanced in comparison to normal borgs.

What about borgs, where would borg-dogs even fit, just a new sprite option, again the sprite even if "fixed" still would not fit, a sameish but different borg that's just a dog, then again non of those things are being proposed in this PR as it is.

> Sci-fi from WH40K to Dr Who has various forms of robotic dogs. I don't see why this is particularly out of theme for the server.

Yes, and yet **I** would not like to see them in the game just the same. Why have borg/robot/synth-dogs when there are already borgs that are pretty much the same exact thing, would NT spend that much money and R&D time to just get a _different_ look for them?

Why not give other borgs, sleeper pods, synth tongues, and boop modules?

Poojawa commented:
Hello. Citadel chiming in. We're the originators of the dogborgs,  Sprites being Verkister. Virgo is bad at accreditation. 

Also these really aren't meant for y'all,  least in terms of the boop/vore etc. They're neat looking but without the medihound Sleeper, it's not the ambulance/emt it was designed for.

K9 is really just an alternative hybrid of Sec and peacekeeper, With a tackle because attack dog. The mobile brig was a Virgo addition.

janitor dog could have the trash compactor swapped with a regular bag, maybe with larger capacity for memes.

Necaladun commented:
@Poojawa thanks!

SamHPurp commented:
Thanks for dropping by Poojawa, while I completely agree that they don't fit with us. Verkister is a damn good spriter, I have to hand it to him.

taukausanake commented:
@Poojawa So nice to hear a voice of reason. Also:

@ExitGame 
> I don't know why you will be looking for spriters if there are practically no volunteers

Me
> I'll try to make free time to mess around with it

So rude

Necaladun commented:
@taukausanake - appreciated! WH40 Arbiters have some great cyberdog stuff. Also anything E-N'ish would be good inspiration.

taukausanake commented:
@Necaladun I want to argue that the double wide sprites are not _bad_ like what everyone else is saying. They don't clip as bad as they say. It would be tough to get anything good looking in 32x32 while also not getting mixed up with current borgi-like pets. That's my first worry. The other is that I'm a shit spriter but I'm willing to help. Recommendations after my concerns?

Necaladun commented:
@taukausanake - double wide has a variety of problems, so you're best off trying to avoid it, although I personally won't complain if it's say, 42*32 or so...

For spriting I have two pieces of advice - dont make it shit and ask people who are better at spriting than I

SomanBat commented:
Leaving aside the whole vore bit, and....this is difficult, the sec dog is quite abit OP even compared to a standard sec borg, and standard sec borgs are pretty rough.  Addressing each part...

1. Jaws are a combat module. Why are we adding a free lethal option to a borg that already has ranged and melee stun, along with the harmbaton?

2. A free reagent scanner, which is hedging in on the Detective's job.  take someone down, strip em, and oh look let me check this syringe without even having to call someone over.

3. Eating trash and batteries for a recharge. One of the very slim nerfs sec borgs have is the need to recharge.  All this will do is extend the "endurance" of the borg, keeping it on the valid hunt even longer.

4. The pounce. Another ranged stun, on top of the disablers.

5. The "portabrig".  This is stupidly powerful for sec to have.  As is, there's a chance to bust free as an antag if caught and cuffed by sec.  Freedom implants, cling armblading, soliciting greytide/thralls to drag you off.  The "portabrig" is a bucklecuff shitcurity combo.  Freedom implant, well you still got to resist free.  Can't get a buddy to drag you away, cuz you're literally inside a thing with multiple combat options.

If sec borgs are a direct upgrade for organic sec, this is a direct upgrade for sec borgs.  Yes, there are plenty of creat sci-fi ideas for robodogs, and lots of directions it can be taken that aren't just "upgraded sec borg".  Idea floated, a bloodhound dogborg that helps the detective track criminals.  A fully combat option attack dog that can only take direct commands from one handler.  Things like that would work way better than what is proposed.

ZomgPonies commented:
I really like the idea of the detective having a bloodhound, player-controllable pet with abilities. You want more doggos? There's a great idea for more doggos that fit our theme, balance and aren't in any way controversial.

Vivalas commented:
I think what is painfully obvious here should be painfully obvious to anyone who isn't pretending or politicking.

Even without the vore associations the concept, in my opinion, is that having borgs with sleepers are bad. We're not against having dog borgs, we're against having dog vorgs that eat people.

I agree with ZomgPonies, something creative like that would be neat. A detective borg would be a nice addition, and making it a bloodhound would be cool. That's a possible idea if you want to salvage this PR from the depths of hell.

taukausanake commented:
Maybe it would be better for dogborgs to be melee based borgs instead. No ranged, maybe the pounce but directional only, not a target thing.
Eating batteries isn't so bad, it's just a faster way of getting them changed out and I see it as a feature. FEED the dog. Maybe it can poop out discharged cells afterwards. :P Other wise if they destroy the cell you're not going to feed them power cells that are higher cap. Waste of materials.
Maybe remove baton and just make their jaws an all purpose tool but only one at a time? Like grab intent.
I was thinking no to the reagent scanner but then @ZomgPonies said "bloodhound" and I was all, "Yeah we should keep it."
No portabrig. It's just too silly even if not vore-y. Maybe the same for janidog trashing someone but the sleeper for a medidog might work. Attach a mech sleeper as a module to them maybe?

Here's a crazy idea: Why let these be borgs at all in that sense? I mentioned once before that being dogs they could have a master along if not instead of the AI. Like a pAI is bound to a master. And add to that, what if they get a new construction method? A dog chassis and get installed into one of the three models? Don't make them borgs, make them something special and a bit more rare

SamHPurp commented:
@ZomgPonies copy pasting what I wrote into Featurechat:

a new simple_animal for the Detective. Possibly a blood hound or an alsatian (another non-dog animal is fine, just historically police and security utilise canines) - however what they get it the ability to sniff scents on things. 

What this does is pops up a 10 second arrow, like a pinpointer, pointing toward the last person that touched that item using last_touched. To stop them from just re-sniffing every 10 seconds, using `last_touched` stops them from being able to bring it with them.

Adds some RP to the Detective roll, adds a slightly more meaningful ghost role

I know there were considerations on adding a second detective... This'd help with that, while adding something more interesting than 'yet another detective'

> "goddamn it, Doyle, find the changeling.... oh god damnit leave Runtime alone!"

Could go further by having each persons scent be a unique hash... so the dog will recognise a scent... so the toolbox he found will smell the same as the jumpsuit worn by `Jonah Bright`...

A lot of cool posibilities.

Vivalas commented:
I like that idea but there would be balance considerations to take into account for sure. Maybe a range or the ability to remove your scent with a bath or something? Otherwise every antag would be screwed as soon as the bloodhound got on their trail, or with that other suggestion, changing their clothes etc.

SamHPurp commented:
@Vivalas de-odorant, taking a shower... ;)

Difrit commented:
This is fucking cancer

Aurorablade commented:
@Poojawa my apologies you were not creditied right.

Spacemanspark commented:
@Difrit refrain from unnecessary/ hostile comments.

Da-Dman234 commented:
Why is this still even being considered? I'm not one to usually pay attention to down/up votes, but holy shit. 2 up to 25 down? It's fine to override the playerbase every once and a while, as I don't think this is a democracy, but ignoring a clear majority here is only going to serve to light more fires and create more drama. (Read: What happened with Vulpakin) People will never be able to get over the fact this is being ported from VORE, as sad as that is to say. Is it honestly even worth the trouble at this point? 

Sorry if that really isn't considered constructive but I believe it's a valid point.

Necaladun commented:
@Da-Dman234 WIP. This is a WIP. WIP. When it's ready to go, we'll look at reactions then that aren't kneejerk reactions about something being from vore.

Necaladun commented:
Reminder: Enough warnings have been given. Shitposting, general attacks, and the like, are not welcome. Be civil and constructive, or you'll just outright be removed.

shazbot194 commented:
How about instead of solo acting dogs, we try to add in borgs that need a helper? I think a medical sleeper that needs someone else to put people into a sleeper (even if its another borg) would be quite cool, and the needing another person negates any problems with it being OP since it would at the point be acting like a mobile sleeper/upgrade sleeper. 
Here is a really bad mock up I did of a syndi version.
![fwsqhudvrvglvdgaoef8fq](https://user-images.githubusercontent.com/22019537/27307994-fd48903e-54f8-11e7-91fe-6b80ce96afbd.png)

MrFroztee commented:
@shazbot194 I like that concept mockup, as it hardly has any features resembling a beast, while getting the point across it's a portasleeper. 

I personally think the borgs we have need a rethink before we go even further complicating things, let alone with a borg/mech hybrid shaped like an animal.

Da-Dman234 commented:
I am 100% there are some people that will never get over the fact that this is from Vore. I only see this as adding fuel to a fire.

These will be both incredibly difficult to balance and take quite a bit of work to even start to get to the point where knee jerk reactions aren't immediate (if there even is a point).

I just don't see it being worth it for all the drama it's caused. I still have yet to see a reason as to WHY we need this, especially with the Peacekeeper borg PR still open

ProperPants commented:
Rather than doing what a good PR does by filling a new niche or improving something that already exists, this PR ports something which makes existing features (the med, sec, and jani borgs) obsolete.

The secdog is objectively better than the secborg, the meddog is worse than the medborg, and the janidog is essentially identical to the janiborg.

All of them having a tongue makes them all better than a janiborg.

All of them having jaws makes them fighting units.

The boop module is as strange, out of place, and ridiculous as the Peacekeeper's cookie dispenser and hug module (and everything else about the Peacekeepers, really).

The question comes: _What's the point?_ You'd be better off finding someone to sprite new 1x1 dog skins for all normal borgs (maybe in the style of E-N, similar to how there is a pAI corgi form), and opening a new PR to add the portabrig, medical pod, and trash compactor.

GeneralGoose commented:
so far the only positive to this dog shit is LE ROBOT WITH SLEEPER!!!!!11 XDD
odysseus can do literally that i bet they even ripped the code from there
u just have to put a mmi into it but people dont use it cuz they say its broken
how about u fix odysseus instead 
its also harder to build so it balances the fact that its an op feature as some say

challade commented:
If you really want to add this you should consider the fact that some people just don't want to see anything... "vore" like. Instead of adding a jiggling belly, make the sprite stay as it is while transporting a person, the inflated and bouncing stomach is what probably turns off most people. 
Also, why is it a wolf chassis? It's strange to transport a human, especially if injured, in the "stomach" of a mechanical wolf. Perhaps the meddog should be made into a "white widow", a medborg in a spider chassis. it carries the human in the abdomen and glows with a green + sign instead of  black widow's red hourglass.
 The secdog is great(K-9), but the janidog doesn't make any sense, maybe something that isn't a dog? A janiborg in a snail chassis would be amazing. It's a snail borg that leaves water as it moves if it would be funny, though it'd make sense if it didn't pull through.

Aurorablade commented:
The sprites are going to be modified I cannot promise a redo of the dogs. I will work on getting them resized but the sleeper sprite issue is going to be handled. Cause kluys is a God panda

SamHPurp commented:
> WIP. This is a WIP. WIP. When it's ready to go, we'll look at reactions then that aren't kneejerk reactions about something being from vore. - @Necaladun 

If thats the case, this should be closed with a new PR when its ready for new review.... Better for the author, better for a fresh set of 'emoji reactions and replies'.

Varghh commented:
Why? Why we need this so bad? 
We have already quite a reputation and i see nothing good adding this up, is wasting a lot of time and work in something we don't need and it has not a single logical reason to exist. We have A LOT of betters things to do and take care than adding dog borg, i really see no reason at all and i'm totally against this.

taukausanake commented:
> this should be closed with a new PR when its ready for new review

But then it'll never be made, which everyone will _love_ but is counter to the reason you put to close it. You can't have a discussion over code if the code doesn't exist and you know the forums won't make a good platform for talking about in-depth code designing

SamHPurp commented:
@taukausanake if you want to gauge new reactions, leaving it here helps no one.

nullbear commented:
the red cross is an issue.

taukausanake commented:
I don't see a red cross. I see a white on red, not the other way around

ExitGame commented:
@taukausanake Have you thougt of that maybe it is better if it doesn't get added?

Ataman commented:
I'm not seeing this merged anytime soon.
'Dog Borgs' as an idea ain't that bad but this thing needs a complete overhaul in my opinion:

1. The sprites, as others pointed out, don't fit Paradise and 64x32 really is an akward format. Redo them completely for 32x32. Make them smaller than people as well. They're supposed to be dogs, aren't they?
2. The features are indeed overpowered and/or useless and need a complete rework. Police tape, really? I suggest taking the standard borg and refining from there. For example: Add additional sensors or other parts to the skeleton (higher resource requirement) and give the dog additional speed in return. Or use the same skeleton (with 4 legs) but give them less HP/Armor for the speed they get.
Remove the ability to cuff people but instead give them a dragging/holding ability which is better than the standard grab mechanic. (Think of pulling but the person being pulled has to resist first before being able to run away). Finetune those features so they make a distinction instead of the dog just being a better borg.
3. Start coding instead of just copying. Instead of a sleeper belly code something new instead. How about a stasis field instead which has the same characteristics but needs to be pulled by the dog? **Be creative**.

Vivalas commented:
I would like to note secborgs already have police tape, if you were labeling that as silly or useless or overpowered or whatever.

Ataman commented:
A dog (robot or otherwise) pulling out police tape and putting it on doors with its paws is silly. The usual borgs have arms for that. Dogs do not.
Also: K9 dogs are trained to attack and restrain a perpetrator. That's what people picture in their mind when they see a police dog. What they certainly don't picture is a dog rolling out the police tape. Doesn't have anything to do with being overpowered. It's simply doesn't fit a dog.

Aurorablade commented:
@Ataman disnpenser on its snout? Bluespace?

REAGARDLES!

Greetings people of the comments, before i go to work and adult let me go over something:
the belly is going to removed and replaced with something like this https://puu.sh/woLiQ/e800294c3a.png thanks to @Bram961 
it will be 'pulled' behind the borg, but will still be a sprite overlay so people can't grab it an run off. I will try to put in safechecks in the code. the inspiration being a st. bernard pulling a sled.
I HAVE found someone to one tile the sprites. BUT he would like me to balance the borgs first before due to the sprite work.

Heres where you guys come in, I could sit here and add remove modules all day but it might not be good so please after reading my big list below suggest what i can remove on the loud out..add to the loud out..or code fresh to add something new. I have poked at the k9 sec unit the most ATM. but i would like to hear your thoughts and inspirations!
I am not going for perfection mind you.

LIST OF ALL THINGS DOGBORG LOADOUT START!:
Added to borg sleepers:
Chance to force person in sleeper out on damage to borg. If i can get working. Grab intent to pull them out
Remove self cleaning

SecHound:
Vice Jaws(Bola legcuff from melee)
Combate jaws(10 damage, can make stamnina)
Forensic nose - detective scanner without report print
Police tape.

Should remove?:
Tongue
Portabrig

Medihound:
Jaws (5 damage, mainly to save paitent from like..spiders)
Nose
Sleeper
Health Analyser
Fire Extinuisher
Borg Hypo
Syringe
Beaker

Should remove?:
Tongue
Extinguisher?
Jaws?
Syringe/beaker?

Janihound:
Light replacer
Holosigns
Nose
Trash Compactor
Tongue (it actually make sense for this borg...)

To change: 
Toggle clean on walk with powe drain for manual tongue floor cleaning?
Why are janiborgs the odd ones...

Ataman commented:
Bluespace technology for police tape? Sounds like a waste of resources which NT would never allow lorewise.
The tape coming out of its snout... Why not, at least it renders all perpetrators unconscious when they're about to die from laughing their asses off.

Are you reworking the dog sprite as well? Or will we now get a 96x32 sprite running around the hallways? Looking up or down the hallway will be veryyy interesting with a 3 tile long sprite.

And why oh why does it have to come in 3 variations with a dozen modules right off the bat. Start small, check the balance and feedback, then work up from there. Simple, really.

Aurorablade commented:
@Ataman the sprite will be one tile, the condition from the spriter is i balance the Borg first.  i am posting for feedback on change and remove of loadout modules. All borgs have a loadout of modules. the concern has been ;why pick this over that' i am trying. I am trying to do the balancing on my own but i would like other peoples thoughts.

Vivalas commented:
I really th

Vivalas commented:
I really think if you are adding a pullable sleeper, it should actually be a physical object, not an overlay which seems somewhat snowflake from a code perspective. Think of it as a downside perhaps.

SamHPurp commented:
I mean... if you want a pullable sleeper, perhaps upgrading the Paramedic's cart to being a cart with a re-usable stasis bag type thing might be cooler, more thematic and gives the role some love.

Aurorablade commented:
Well the current sleeper is attached to the sprite and as an overlay this would do the same. As for pullqble object it's possible but quirks might happen with missing borg modules if it's stolen. Anything else for medihound

Ataman commented:
Wouldn't a sprite overlay look really awkward when turning? Since it's hooked to the sprite itself the cart would skip over a tile evertime you change directions.

Aurorablade commented:
Yeah I will just have to see what one works the the best. I need to get balance down

Aurorablade commented:
I may have to use the current sprites to judge as I said I resized sprites come when I get balance done I like how the K9 is shaping. I now need EMT Borg and the odd duck of a janiborg

Tayyyyyyy commented:
I really like the idea of a search and rescue dog. What if instead of a dog borg, Ian could be fitted with a suit and internals, and then trained to do search and rescue?

optimumtact commented:
please merge this so we can finally settle the question about if /tg/ or paradise has the moral high ground

Aurorablade commented:
@Tayyyyyyy we could always give ian a new costume.

Aurorablade commented:
It is evident from this, and the previous, PR, that the response to the concept is incredibly negative; it would require, in essence, a complete rework to even be feasible. Therefore, it is being closed.

Qustinnus commented:
thanks dad

mrturkeytoe commented:
![image](https://user-images.githubusercontent.com/16676120/27465315-5b84e912-5798-11e7-80bf-6b40ea6ac39a.png)
